public/sitemap.txt

```diff
@@ -1,11 +1,9 @@
 https://app.shapeshift.com
-https://app.shapeshift.com/#/connect-wallet
-https://app.shapeshift.com/#/demo/fox
-https://app.shapeshift.com/#/demo/defi
-https://app.shapeshift.com/#/demo/defi/earn
-https://app.shapeshift.com/#/demo/trade
-https://app.shapeshift.com/#/demo/assets
-https://app.shapeshift.com/#/demo/assets/bip122:000000000019d6689c085ae165831e93/slip44:0
-https://app.shapeshift.com/#/demo/assets/eip155:1/slip44:60
-https://app.shapeshift.com/#/demo/assets/eip155:1/erc20:0xc770eefad204b5180df6a14ee197d99d808ee52d
-https://app.shapeshift.com/#/demo/assets/eip155:1/erc20:0xdc49108ce5c57bc3408c3a5e95f3d864ec386ed3
\ No newline at end of file
+https://app.shapeshift.com/#/fox
+https://app.shapeshift.com/#/rFOX
+https://app.shapeshift.com/#/trade
+https://app.shapeshift.com/#/buy-crypto
+https://app.shapeshift.com/#/assets/bip122:000000000019d6689c085ae165831e93/slip44:0
+https://app.shapeshift.com/#/assets/eip155:1/slip44:60
+https://app.shapeshift.com/#/assets/eip155:1/erc20:0xc770eefad204b5180df6a14ee197d99d808ee52d
+https://app.shapeshift.com/#/assets/eip155:1/erc20:0xdc49108ce5c57bc3408c3a5e95f3d864ec386ed3
```

Comment:
q: I don't know if it's useful but we can trade and pairs here, so we could index more pages with different pairs (maybe it would rank us up in SEO?)

Reply:
The more routes we add, I believe the better! Can't be worse than currently.

<img width="1371" alt="image" src="https://github.com/user-attachments/assets/9ea1c48f-da89-43c5-a1c9-bce323178ae8" />


On a tangent re: SEO, I think we could also:
- find a way to make title head work, not sure if we can do this without SSR, but all pages being references
- ensure our crawling is sane and scales - would be nice to have a pass on that in a follow-up

If you want to tag team with hpayne and own this, feel free to create an issue and bring it to the table! 💪🏽

Reply:
Pretty sure we can without SSR, maybe it would require some page generation or something similar

Added to my todo, I'll think about it and create an issue depending on what I find!